tier1-icons.html

```diff
@@ -40,8 +40,7 @@
           <path d="M17.707 5.707l-8 8a1 1 0 0 1-1.414 0l-8-8A1 1 0 0 1 1 4h16a1 1 0 0 1 .924.617A.97.97 0 0 1 18 5a1 1 0 0 1-.293.707z"/>
         </g>
         <g id="arrow-expand-small" mirror-in-rtl="true">
-          <path d="M6,14V6c0-0.4,0.2-0.8,0.6-0.9S7.4,5,7.7,5.3l4,4c0.4,0.4,0.4,1,0,1.4l-4,4C7.4,15,7,15.1,6.6,14.9
-	C6.2,14.8,6,14.4,6,14z M8,8.4v3.2L9.6,10L8,8.4z"/>
+          <path d="M6,14V6c0-0.4,0.2-0.8,0.6-0.9S7.4,5,7.7,5.3l4,4c0.4,0.4,0.4,1,0,1.4l-4,4C7.4,15,7,15.1,6.6,14.9&#xA;&#x9;C6.2,14.8,6,14.4,6,14z M8,8.4v3.2L9.6,10L8,8.4z"/>
```

Comment:
Was this meant to change?

Reply:
No, but it happened as a result of `npm run build`, and so I thought it was normal?

Reply:
I think it's OK... probably a result of the last person _not_ using the generated script?